Assets/MixedRealityToolkit/InputModule/Prefabs/Cursor/InteractiveMeshCursor.prefab

```diff
@@ -212,56 +187,6 @@ MeshFilter:
   m_PrefabInternal: {fileID: 100100000}
   m_GameObject: {fileID: 1000013883439490}
   m_Mesh: {fileID: 4300002, guid: d6fab9b92d520f54fb1d447be32320f2, type: 3}
---- !u!108 &108000011815251248
-Light:
-  m_ObjectHideFlags: 1
-  m_PrefabParentObject: {fileID: 0}
-  m_PrefabInternal: {fileID: 100100000}
-  m_GameObject: {fileID: 1000012327360030}
```

Comment:
Why was this GameObject deleted from the prefab?

Reply:
This was the Unity light on the cursor, which was replaced with a HoverLight. I'll fix the missing reference above and make sure if can be toggle on/off like the InteractiveMeshCursor requires. Thanks for pointing this out!